src/views/Ifos/hooks/v3/useGetWalletIfoData.ts

```diff
@@ -74,20 +81,68 @@ const useGetWalletIfoData = (ifo: Ifo): WalletIfoData => {
       params: [account, [0, 1]],
     }))
 
+    const IfoV3Contract = getIfoV3Contract(address)
```

Comment:
`const contract = useIfoV3Contract(address)`
same contract above?

Reply:
Ya, is same. But i can't get read data from `const contract = useIfoV3Contract(address)`